packages/itinerary-body/src/AccessLegBody/index.tsx

```diff
@@ -111,38 +114,47 @@ class AccessLegBody extends Component<Props, State> {
           <S.LegDetails>
             <S.StepsHeaderAndMapLink>
               <S.StepsHeader
-                aria-expanded={expanded}
-                onClick={this.onStepsHeaderClick}
+                aria-expanded={!hideDrivingDirections ? expanded : undefined}
+                as={hideDrivingDirections && "span"}
+                autoCursor={hideDrivingDirections}
+                onClick={!hideDrivingDirections && this.onStepsHeaderClick}
```

Comment:
Please use a `?` ternary to separate the button mode vs. the span mode, so that we don't end up with all of these conditional props around.

Reply:
Much cleaner, thank you! Fixed in [53a5c5f](https://github.com/opentripplanner/otp-ui/pull/658/commits/53a5c5f9b01fdd1b78e0a02d13743805c600454b)